### PR TITLE
Composer require civicrm_entity ~3.0

### PR DESCRIPTION
**Pull request by Justin-Fernbaugh**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
         "civicrm/civicrm-packages": "~5.41",
         "civicrm/cv": "~0.3",
         "composer/installers": "~2.2",
+        "drupal/civicrm_entity": "^3.0'",
```
Comment (ramereth) [inline composer.json:24]: Try using `~` instead. Also, you need to update the `compose.lock` (see README.md)

**Review by ramereth — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (Justin-Fernbaugh) [reply composer.json]: Just pushed those changes

**Comment by ramereth** [comment]
@Justin-Fernbaugh can I assume this PR shouldn't be merged until this [PR](https://github.com/osuosl-cookbooks/proj-snowdrift/pull/103) is merged as well?

**Comment by Justin-Fernbaugh** [comment]
> @Justin-Fernbaugh can I assume this PR shouldn't be merged until this [PR](https://github.com/osuosl-cookbooks/proj-snowdrift/pull/103) is merged as well?

I don't believe merging this PR first would produce any errors but yes, they should probably be merged together.

**Review by ramereth — APPROVED** [review]

**Comment by Justin-Fernbaugh** [comment]
!bump patch ~

**Comment by ramereth** [comment]
@Justin-Fernbaugh you can just merge this :)

**Pull request merged**